### PR TITLE
Update `x11-clipboard` to `0.9.1`

### PR DESCRIPTION
**Pull request by fornwall**
This updated the minimum supported rust version on X11 from `1.65` to `1.66`.

If possible, it would be nice to publish a minor release with this update, as it resolves some duplicated dependencies when using `copypasta` together with latest `winit 0.29`.

### CODE REVIEW

**Comment by fornwall** [comment]
> This updated the minimum supported rust version on X11 from 1.65 to 1.66.

Actually, due to an issue the current MSRV is 1.74 - let's await https://github.com/quininer/x11-clipboard/pull/47 fixing that.

**Comment by fornwall** [comment]
> Actually, due to an issue the current MSRV is 1.74 - let's await https://github.com/quininer/x11-clipboard/pull/47 fixing that.

That has now been resolved and release, and this PR is ready for review @chrisduerr.

File: .builds/freebsd.yml [hunk 1]
```diff
@@ -25,5 +25,5 @@ tasks:
       cargo clippy --all-targets
   - oldstable: |
       cd copypasta
-      rustup toolchain install --profile minimal 1.65.0
-      cargo +1.65.0 test
+      rustup toolchain install --profile minimal 1.66.0
+      cargo +1.66.0 test
```
Comment (chrisduerr) [inline .builds/freebsd.yml:29]: MSRV bumps should be split into a separate commit (before the actual change obviously).

**Review by chrisduerr — CHANGES_REQUESTED** [review]

**Comment by kchibisov** [comment]
Well, I can't really test myself, so it's up to you @chrisduerr .

**Comment by chrisduerr** [comment]
> Well, I can't really test myself, so it's up to you @chrisduerr .

Just requested your review in case you're aware of anything noteworthy with this release.

**Comment by kchibisov** [comment]
They will remove the last duplicate from alacritty deps ¯\_(ツ)_/¯ and also fixed something.

File: .builds/freebsd.yml (hunk 1 above)
Comment (fornwall) [reply .builds/freebsd.yml:29]: Thanks, done!

**Comment by fornwall** [comment]
> They will remove the last duplicate from alacritty deps ¯_(ツ)_/¯ and also fixed something.

Removing duplicated dependencies was the main driver for us as well. Though getting https://github.com/quininer/x11-clipboard/pull/46, a fix for a connection leak that was seen, is also nice.

**Review by kchibisov — APPROVED** [review]

**Review by chrisduerr — APPROVED** [review]
